### DISCUSSION TITLE
Server does not detect unlicensed plugin without rescan and can crash

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#458] [Help]
I just updated to AG 1.1.1 but I can't seem to even start or load a project. It crashes right away.

**Comment by apohl79** [comment]
https://audiogridder.com/bug-reports/

**Reply by jaacMusic** [reply]
Hello Andreas, I think I found out the issue. If a plugin is not licensed for some reason but is detected as an installed VST plugin it would cause a crash as the AG plugin or server can't load the plugin. The only way I found about it is that installed a music app to test the installation of the VST and I found out it wasn't registered/licensed. After registering offensive VST it worked fine!
Maybe, you could have some error handling for this type of scenario.

Thanks!!

**Reply by apohl79** [reply]
Understood. That makes sense. I will have a look at this and create a ticket. Thanks for reporting back!

**Reply by emack999** [reply]
@jaacMusic Perhaps change the title (if you can ) to **_AG 1.1.1 crashes from unregistered plugin_** (or something like that).

**Comment by CaBuffalo** [comment]
The answer is yes or no. I did register of my IK plugin in AG before. Logic
and S1 didn't crash.

On Tue, Mar 30, 2021 at 3:50 PM Andreas Pohl ***@***.***>
wrote:

> Understood. That makes sense. I will have a look at this and create a
> ticket. Thanks for reporting back!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/458#discussioncomment-550513>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6EBHFNX2CB3XJKXZBDTGJIVDANCNFSM42DBPWYA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by jaacMusic** [reply]
I retried the same scenario on my backup PC and the same happened again. So I can confirm that some VST if they are not licensed would crash AG.
